.github/workflows/cli-release.yml

```diff
@@ -65,6 +65,8 @@ jobs:
 
           if [[ ${{ env.PUBLISH_NIGHTLY }} == 'true' ]]; then
             export NIGHTLY="--snapshot"
+            LATEST_TAG=$(curl -s https://api.github.com/repos/rilldata/rill/releases/latest | jq -r '.name')
+            echo "GORELEASER_CURRENT_TAG=${LATEST_TAG}" >> .release-env 
```

Comment:
Any way to do this using `git` commands or built-in features in either Goreleaser or GH actions? Wondering if this might be fragile given it's an external network request and relies on a specific response format.